services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -102,6 +101,7 @@ public void configure(Binder binder)
 
             binder.bind(JettyServerInitializer.class).to(QueryJettyServerInitializer.class).in(LazySingleton.class);
             Jerseys.addResource(binder, QueryResource.class);
+            Jerseys.addResource(binder, QueryResourceV3.class);
```

Comment:
I believe this needs to be registered for router, historical, RT Index task and RT Nodes as well ?